Makefile

```diff
@@ -1,6 +1,11 @@
-# minimal Makefile to run Python unit tests
-# for flux-accounting
+FLUX_INSTALL_PREFIX ?= /usr
+
+dependencies:
+        ${FLUX_INSTALL_PREFIX}/bin/flux python -m pip install -r requirements.txt --user
```

Comment:
The beginning of these "recipe" lines (lines meant to be run in the shell) should start with a `<TAB>` rather than spaces [[1]](https://www.gnu.org/software/make/manual/html_node/Recipe-Syntax.html).  It's an oddity of Makefiles that has also bitten me in the past:

```
sherbein ~/Repositories/flux-framework/flux-accounting (add-make-install)
❯ make install                                                                                                              16:25:21 ()
Makefile:4: *** missing separator (did you mean TAB instead of 8 spaces?).  Stop.
```